### ISSUE TITLE
The model in the Readme may be misspecified

### ISSUE THREAD

**Issue by mikemc** [zhaoni153/MicroBias#2]
The model shown in the Readme right now is `$\ln \tilde p_{ij} = \ln p_{ij} + X_{i\cdot} \beta_{\cdot j} + \epsilon_{ij}$`, which is missing the sample-specific normalization term `$\alpha_i$` from the manuscript and I think would be needed for the proportions to sum to one under the MWC model.

**Comment by zhaoni153** [comment]
Indeed, there is an error. Have corrected now. Thanks for pointing out.
